docs/handbook/tutorial.rst

```diff
@@ -471,14 +471,15 @@ Reading from an open file
 To read an image from string data, use the :py:class:`~StringIO.StringIO`
```

Comment:
Should this also be changed?

Reply:
:+1: thanks. Fixed in the latest revision.